src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,34 @@
+.top-bar {
+  padding-block: 24px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @include onTablet() {
+    padding-block: 32px;
+  }
+
+  &__icons {
+    display: flex;
+    gap: 24px;
+
+    @include onTablet() {
+      gap: 32px;
+    }
+  }
+
+  &__logo-link {
+    display: flex;
+  }
+
+  &__logo {
+    height: 16px;
+
+    @include hover(transform, scale(1.2));
+
+    @include onTablet {
+      height: 24px;
+    }
+  }
+}
```

Comment:
I don't see the phone number on the hover

Reply:
I don't really understand what should I do. I have <a href="tel:...">. What should I add? In the bottom-left angle of the page we see the phone number on hover.